plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -429,6 +432,41 @@ export class AwsLambdaInstrumentation extends InstrumentationBase<AwsLambdaInstr
     return propagation.extract(otelContext.active(), httpHeaders, headerGetter);
   }
 
+  private static _extractOtherEventFields(event: any): Attributes {
+    const answer: Attributes = {};
+    const fullUrl = this._extractFullUrl(event);
+    if (fullUrl) {
+      answer[SEMATTRS_HTTP_URL] = fullUrl;
+    }
+    return answer;
+  }
+
+  private static _extractFullUrl(event: any): string | undefined {
+    // API gateway encodes a lot of url information in various places to recompute this
+    if (!event.headers || !event.path) {
```

Comment:
For API Gateway Proxy event, unlike REST API, HTTP API doesn't have `path` property, but `rawPath` property.

Reply:
https://docs.aws.amazon.com/lambda/latest/dg/services-apigateway.html#apigateway-example-event

Reply:
TIL, thanks.  I'll adjust the code to look for either and add a corresponding test.